### ISSUE TITLE
Big picture is under the close button

### ISSUE THREAD

**Issue by Faelian** [alexgleason/elementary-steam-skin#1]
Hello,

That's really a cool skin, but I have notice a glich on freya with steam installed from the repository :
![big picture glitch](https://cloud.githubusercontent.com/assets/4268373/11170723/c578b382-8bdc-11e5-8de8-f7b862fbc240.png)

The button for big picture is under the close button.
Which make it hard to reach, and it's easy to click on it instead of close.

**Comment by alexgleason** [comment]
Thanks for pointing this out! The skin may be outdated at this point. I haven't played games in a while and don't currently have Steam installed on the machine I'm typing on. I can check this out next time I'm running Steam, but if you have any spare time I encourage you to help! I added you as a collaborator to this project. Thanks. :smile:

**Comment by sergioad** [comment]
Hello I am having the  same exact issue

**Comment by alexgleason** [comment]
Yep, it's confirmed. The skin it outdated. It's because Steam recently revamped their UI. This skin is broken and needs to be fixed.

**Comment by sergioad** [comment]
yes, indeed

**Comment by sergioad** [comment]
@alexgleason how is the fix going? there is a skin named Metro for steam wich can be used as the base for a next release of it, its name is metro for steam and it is awesome, it decorates the window with a GNOME / eOS like headerbar and turns the menubar into a single menu, again it is perfect for the base of the skin

**Comment by alexgleason** [comment]
Looks like there's ongoing development here. https://github.com/iXecFTW/steam_ElementaryOS
